db/migrate/20160927230136_enable_fk_message_digests_nodes.rb

```diff
@@ -0,0 +1,21 @@
+class EnableFkMessageDigestsNodes < ActiveRecord::Migration
+
+  class Bag < ActiveRecord::Base
+  end
+
+  class MessageDigest < ActiveRecord::Base
+    belongs_to :bag
+  end
+
+  def up
```

Comment:
Let's assume that *135 already ran, then it should be the case that *134 would run (to remove any foreign key, named or not), then *135 would not run (but the effect of having a NOT NULL node_id would be in tact), and then *136 would add the named foreign key.  Is there some reason *134 would not run, even if *135 has already run?  I think the migration just iterates over them, assuming they are idempotent up/down/change ops (or should be).